### ISSUE TITLE
Dailymotion videos no longer play

### ISSUE THREAD

**Issue by xRyozuo** [calzoneman/sync#980]
**Please confirm whether you've tried the following debugging steps:**

  - [X] Clearing cache and refreshing the page (On Firefox, press Ctrl+F5.  On Chrome, press F12, then right click the refresh button and click "Empty Cache and Hard Reload")
  - [X] Disabling all browser extensions
  - [x] Using a clean channel with no customizations
  (also tried on several different devices and browsers)
  
### Description of the Problem ###
Videos from dailymotion dont seem to play anymore (tried adding new random videos, nada), this started happening sometime this week. The channel had been running and working for a few months with no issues. Now theres a blank black box in its stead but the header does display the correct title of the video.

### System Information ###
CONFIRMED IT DOESNT WORK ON:
  - **Operating System (Windows / iOS):**
  - **Web Browser (Firefox / Safari / Edge):**
  - **Error Messages Displayed:** N/A
  - **Screenshot of JavaScript Console:**
 unreachable code after return statement
[all.js:2:23140](https://api.dmcdn.net/all.js)
downloadable font: Glyph bbox was incorrect (glyph ids 27 29 124 127 128 136 149 150 156 162 181 198 201) (font-family: "Glyphicons Halflings" style:normal weight:400 stretch:100 src index:1) source: https://maxcdn.bootstrapcdn.com/bootstrap/3.3.1/fonts/glyphicons-halflings-regular.woff
Connecting to {"url":"https://zip.cytu.be:8443/","secure":true} [callbacks.js:1256:13](https://cytu.be/js/callbacks.js)
This integration method is deprecated. We advise using Player Embeds https://faq.dailymotion.com/hc/en-us/articles/4411096679954-Integrate-your-Player-Embed

**Comment by calzoneman** [comment]
Hi, thanks for reporting this.  I'm no longer actively maintaining this software so we'll have to see if someone else is willing to redo the dailymotion implementation as it looks like they changed it.

**Comment by xRyozuo** [comment]
Oh shame! So out of curiosity, what would I need to learn to do this
myself? I depend on the channel to fall asleep so I’m motivated lol

**Comment by Xaekai** [comment]
The documentation for DailyMotion player APIs is here:
https://developers.dailymotion.com/sdk/player-sdk/web/#methods

The page referenced in the console errors of CyTube's webclient when you play a DailyMotion video current is here:
https://developers.dailymotion.com/guides/getting-started-with-web-sdk/#player-library-script


I do my best to keep CyTube running but to be quite honest, DailyMotion is something I use quite rarely, and I have other more pressing concerns with the site, so it would be a long time before I could do this. If you're interested in doing it yourself, you'd need to know basic JavaScript, and to look at how the YouTube integration is done. You'd need to embed the DM player library, and modify CyTube's player class for DM.
https://github.com/calzoneman/sync/blob/3.0/player/dailymotion.coffee

**Comment by xRyozuo** [comment]
Thank you very much for the roadmap! Ive been tinkering with it, as i understand it i need to create a fork to edit the file. How can i test that what Im doing works/doesnt before uploading it?

**Comment by frdrck** [comment]
I'd love to see this fixed, dailymotion is pretty integral for 480p media

**Comment by hdfssk** [comment]
I've got Dailymotion working again locally, and opened a PR here: https://github.com/calzoneman/sync/pull/983

**Comment by Xaekai** [comment]
The changes by @hdfssk were deployed but don't seem to address this.

**Comment by xRyozuo** [comment]
Hello, by chance i checked out the old dailymotion channel i had and it seems to be working now. Unsure if it was a change by someone on here or a change in dailymotion, but this issue seems to have been solved!

**Comment by Spengreb** [comment]
Yeah i think this can be closed, dailymotion seems to be working again
